AdyenDropIn/DropInComponentExtensions.swift

```diff
@@ -42,7 +42,19 @@ extension DropInComponent: PaymentMethodListComponentDelegate {
             self?.paymentMethods.stored.removeAll(where: { $0 == paymentMethod })
             self?.reloadComponentManager()
         }
-        storedPaymentMethodsDelegate?.disable(storedPaymentMethod: paymentMethod, completion: deletionCompletion)
+        
+        if let sessionAsStoredPaymentMethodsDelegate {
+            sessionAsStoredPaymentMethodsDelegate.disable(
+                storedPaymentMethod: paymentMethod,
+                dropInComponent: self,
+                completion: deletionCompletion
+            )
+        } else {
+            storedPaymentMethodsDelegate?.disable(
```

Comment:
Can this `storedPaymentMethodsDelegate` be nil? What happens if that's the case?

Reply:
Can be nil of course and yeah nothing happens then